metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py

```diff
@@ -60,7 +60,12 @@ def _load_viewfile(
             with open(path) as file:
                 raw_file_content = file.read()
         except Exception as e:
-            self.reporter.failure("Failed to read lkml file", path, exc=e)
+            self.reporter.failure(
+                title="LKML File Loading Error",
+                message=f"Failed to read lkml file {path}",
```

Comment:
the path should be in `context`, not `message`. Message and title should always be constant strings